src/ui/Carousel/tests/Carousel.spec.tsx

```diff
@@ -0,0 +1,106 @@
+import { mount } from '@cypress/react';
+import { composeStory } from '@storybook/testing-react';
+
+import Meta, { CarouselDemo } from '../stories/Carousel.stories';
+
+const CarouselStory = composeStory(CarouselDemo, Meta);
+
+const content = (
+  <div style={{ margin: '5rem 0 5rem 20rem' }}>
+    <CarouselStory />
+  </div>
+);
+
+describe('Carousel', () => {
+  it('Manual mode using left and right button', () => {
+    mount(content);
+
+    cy.get('[data-test="2-image"]').should('be.visible');
+
+    cy.get('[data-test="animals-carousel-left-button"]').trigger('click');
+
+    cy.get('[data-test="1-image"]').should('be.visible');
+
+    // Can't click before transition is done:
+    cy.wait(350);

```

Comment:
Isn't Cypress able to automatically determine if an element is transitioning and wait until it ends?